tests/tests_suite_test.go

```diff
@@ -53,6 +63,20 @@ var _ = AfterSuite(func() {
 	}
 })
 
+func GetApiConnection() (net.Conn, error) {
+	podList, err := ApiClient.CoreV1().Pods(DeploymentNamespace).List(context.TODO(), metav1.ListOptions{
```

Comment:
Will there only be one pod at a time? Maybe assert on that? What about filtering for not running pods (see https://github.com/kubevirt/ssp-operator/pull/478), we had problems with that in the past.

Reply:
Good point, changed.